lib/normalize-contents.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = function(contents) {
+  let hashbangOrNot = '';
+
+  if (contents.startsWith('#!')) {
+    let lines = contents.split(/\n/);
```

Comment:
In the latest version of the proposal, the hashbang is terminated with a LineTerminator--not just a Line Feed. Test262 is also missing tests for other LineTerminators, so [I've submitted a few](https://github.com/tc39/test262/pull/2104)

Reply:
Ah! Good catch. I'll work on updating this.